standalone-integration/src/test/java/com/ericsson/bss/cassandra/ecchronos/standalone/ITOnDemandRepairJob.java

```diff
@@ -70,9 +74,19 @@
 import com.ericsson.bss.cassandra.ecchronos.core.utils.TableReferenceFactory;
 import com.ericsson.bss.cassandra.ecchronos.core.utils.TableReferenceFactoryImpl;
 
+@RunWith(Parameterized.class)
 @NotThreadSafe
 public class ITOnDemandRepairJob extends TestBase
 {
+    @Parameterized.Parameters
+    public static List<Boolean> routingOptions()
+    {
+        return Arrays.asList(true, false);
+    }
+
+    @Parameterized.Parameter
+    public Boolean myRemoteRoutingOption;
```

Comment:
Not used anywhere. Is this needed for the Parameterized tests?

Reply:
They arw needed for the parameterized things to work.